pkg/render/logstorage.go

```diff
@@ -1241,7 +1243,7 @@ func (es elasticsearchComponent) kibanaCR() *kbv1.Kibana {
 			Config: &cmnv1.Config{
 				Data: config,
 			},
-			Count: 1,
+			Count: *es.cfg.Installation.ControlPlaneReplicas,
```

Comment:
You might get a nil pointer exception here - unlikely, since this field should be defaulted, but a safer thing to do would be to check if this field is nil and set `Count: 0` if it is.

Reply:
Add a check for nil pointer and default the count to 1 as before. I think we need to have at least 1 kibana deployment.

Reply:
1 is probably fine. I suggested 0 because it is more obviously an error scenario that can be asserted in tests, etc, but I don't really mind.